network/src/main/java/CertificateError.java

```diff
@@ -0,0 +1,60 @@
+/*
+ *  Copyright 2018, TeamDev. All rights reserved.
+ *
+ *  Redistribution and use in source and/or binary forms, with or without
+ *  modification, must retain the above copyright notice and the following
+ *  disclaimer.
+ *
+ *  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ *  "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ *  LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ *  A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ *  OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ *  SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ *  LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ *  DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ *  THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ *  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ *  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import com.teamdev.jxbrowser.chromium.Browser;
+import com.teamdev.jxbrowser.chromium.Certificate;
+import com.teamdev.jxbrowser.chromium.CertificateErrorParams;
+import com.teamdev.jxbrowser.chromium.DefaultLoadHandler;
+import com.teamdev.jxbrowser.chromium.swing.BrowserView;
+
+import javax.swing.*;
+import java.awt.*;
+
+/**
+ * Demonstrates how to handle SSL certificate errors.
+ */
+public class CertificateError {
+    public static void main(String[] args) {
+        Browser browser = new Browser();
+        BrowserView browserView = new BrowserView(browser);
+
+        JFrame frame = new JFrame();
+        frame.setDefaultCloseOperation(WindowConstants.EXIT_ON_CLOSE);
+        frame.add(browserView, BorderLayout.CENTER);
+        frame.setSize(700, 500);
+        frame.setLocationRelativeTo(null);
+        frame.setVisible(true);
+
+        browser.setLoadHandler(new DefaultLoadHandler() {
+            @Override
+            public boolean onCertificateError(CertificateErrorParams params) {
+                Certificate certificate = params.getCertificate();
+                System.out.println("subjectName = " + certificate.getSubjectName());
+                System.out.println("issuerName = " + certificate.getIssuerName());
+                System.out.println("hasExpired = " + certificate.hasExpired());
+                System.out.println("errorCode = " + params.getCertificateError());
+                // Return false to ignore certificate error.
+                return false;
+            }
+        });
+        browser.loadURL("<https-url-with-invalid-ssl-certificate>");
```

Comment:
Can we arrange such a location and a certificate? It's hard to see the example working without it. Please speak with @vladimir-ikryanov on this.